charts/bungee/templates/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.bungeeCord.config }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "bungee.fullname" . }}-config
+  labels:
+    app: {{ template "bungee.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  config.yml: |
+{{ toYaml .Values.bungeeCord.config | indent 6 }}
```

Comment:
This was a great idea

Reply:
Thanks! This likely needs to just be `{{ .Values.bungeeCord.config | indent 6 }}` due to Velocity's `.toml` format.
There's still the issue that Velocity needs write access to the file or it crashes on start, so I should work out if it's worth supporting that use or attempting to find a solve